vcsclient/bitbucketcommon.go

```diff
@@ -7,7 +7,10 @@ import (
 	"time"
 )
 
-const notSupportedOnBitbucket = "currently not supported on Bitbucket"
+const (
+	notSupportedOnBitbucket     = "currently not supported on Bitbucket"
+	bitbucketPrContentSizeLimit = 32768
```

Comment:
same here

Reply:
I didn't find any API or reference at the library we use. I don't think we can get it